packages/likes/src/likes.tsx

```diff
@@ -19,24 +18,32 @@ type Props = {
       url: string;
     };
     votesNeeded?: number;
+    votes: {
+      isActive: boolean;
+      requiredUserRole: string;
+      voteType: string;
+      voteValues: Array<{
+        label: string;
+        value: string;
+      }>;
+    };
+    login: {
+      url: string;
+    };
   };
 };
 
 function Likes(props: Props) {
-  const projectId = props.projectId || props.config?.projectId;
-  const ideaId = props.ideaId || props.config?.ideaId;
-  const apIurl = props.apiUrl || props.config.api?.url;
   const necessaryVotes = props?.config?.votesNeeded || 50;
 
   const datastore = new DataStore(props);
   const session = new SessionStorage(props);
 
-  const [currentUser, currentUserError, currentUserIsLoading] = datastore.useCurrentUser({ ...props });
-  const [idea, ideaError, ideaIsLoading] = datastore.useIdea({ ...props });
-  const [isBusy, setIsBusy] = useState(false)
+  const [currentUser] = datastore.useCurrentUser({ ...props });
```

Comment:
Waarom wordt hier eigenlijk een spread gebruikt?

Reply:
Geen idee, is inderdaad niet nodig. Is aangepast